addons/mail/wizard/mail_compose_message.py

```diff
@@ -283,6 +279,26 @@ def _inverse_reply_to_mode(self):
         for composer in self:
             composer.reply_to_force_new = composer.reply_to_mode == 'new'
 
+    @api.depends('composition_mode', 'model', 'parent_id', 'res_ids', 'template_id')
+    def _compute_partner_ids(self):
+        """ Recipients computation. When based on template it uses its 3 fields
+        email_cc, email_to and partner_to. All emails are converted into partners
+        (creating new one if not found). Indeed composer does not deal with emails
+        but only with partners. """
+        for composer in self:
+            res_ids = composer._parse_res_ids(composer.res_ids)
+            if composer.template_id and composer.composition_mode == 'comment' and len(res_ids) == 1:
+                rendered_values = composer._generate_template_for_composer(
+                    composer.template_id,
+                    res_ids,
+                    ('email_cc', 'email_to', 'partner_ids'),
+                    partners_only=True
+                )[res_ids[0]]
+                if rendered_values.get('partner_ids'):
+                    composer.partner_ids = rendered_values['partner_ids']
+            elif composer.parent_id and composer.composition_mode == 'comment':
+                composer.partner_ids += composer.parent_id.partner_ids
```

Comment:
same here, shouldn't we add a else and write False to reset the value ? (otherwise if we change the composition mode, it will keep the `partner_ids`)